### PR TITLE
Use .example extension for properties that require user to update

### PR DESCRIPTION
**Pull request by jimmyjames**
**Changes**

- Move `auth0.properties` file to `auth0.properties.example`.
- Update `README` to add instructions to copy `.example` file to `auth0.properties`.
- Also updates `README` to use the term "Application" instead of "Client".

**Reason**

- Developers need to provide their own values for their application, but should not commit those changes to this repository. Using an `.example` extension allows for a source-controlled template that can be copied for real use.

### CODE REVIEW

File: 01-Login/README.md [hunk 1]
```diff
@@ -6,13 +6,20 @@
...
+1. Add the URL that will be called on logout to the Allowed Logout URLs. i.e.: `http://localhost:3000`.
 1. Copy the `Domain`, `Client ID` and `Client Secret` values at the top of the page and use them to configure the Java Application.
 
 
 ### Java Application
+
+Copy `src/main/resources/auth0.properties.example` to `src/main/resources/auth0.properties`:
+
+```bash
+cp src/main/resources/auth0.properties.example src/main/resources/auth0.properties
```
Comment (lbalmaceda) [inline 01-Login/README.md:20]: what if we rename instead?
```suggestion
mv src/main/resources/auth0.properties.example src/main/resources/auth0.properties
```

File: 01-Login/README.md [hunk 2]
```diff
@@ -6,13 +6,20 @@
...
-1. Add the URL that will be called on logout to the Allowed Logout URLs. i.e.: `https://mysite.com`.
+1. On the [Auth0 Dashboard](https://manage.auth0.com/#/clients) create a new Application of type `Regular Web Application`. 
+1. Add the URL that will be called on an OAuth successful login to the Allowed Callback URLs. i.e.: `http://localhost:3000/callback`.
+1. Add the URL that will be called on logout to the Allowed Logout URLs. i.e.: `http://localhost:3000`.
 1. Copy the `Domain`, `Client ID` and `Client Secret` values at the top of the page and use them to configure the Java Application.
 
 
 ### Java Application
+
+Copy `src/main/resources/auth0.properties.example` to `src/main/resources/auth0.properties`:
```
Comment (lbalmaceda) [inline 01-Login/README.md:17]: "Rename" instead of copy?

File: 01-Login/README.md (hunk 1 above)
Comment (jimmyjames) [reply 01-Login/README.md:20]: If we rename, that would show as a modified file in the git working tree, which seems like something I wouldn't want as a developer.

File: 01-Login/README.md (hunk 2 above)
Comment (jimmyjames) [reply 01-Login/README.md:17]: If we rename, that would show as a modified file in the git working tree, which seems like something I wouldn't want as a developer.

Comment (albertoperdomo) [reply 01-Login/README.md:17]: I too think it's best if the local copy is displayed as clean.

**Review by lbalmaceda — APPROVED** [review]

**Pull request merged**